*** uLine.ml.org	Thu Nov 20 16:55:22 2003
--- uLine.ml	Thu Nov 20 17:15:18 2003
***************
*** 132,139 ****
  	  | n -> 
  	      Text.Buf.add_char b (uchar_of_int n); 
  	      loop () in
! 	loop ();
! 	Text.Buf.contents b
  
        method close : unit =
  	Text.Buf.reset b;
--- 132,145 ----
  	  | n -> 
  	      Text.Buf.add_char b (uchar_of_int n); 
  	      loop () in
! 	try
! 	  loop ();
! 	  Text.Buf.contents b
! 	with End_of_file ->
! 	  if Text.length (Text.Buf.contents b) > 0 then
! 	    Text.Buf.contents b
! 	  else
! 	    raise End_of_file
  
        method close : unit =
  	Text.Buf.reset b;

% Makefile.in
*** Makefile.in.org	Sat Sep  6 13:37:21 2003
--- Makefile.in	Thu Nov 20 18:23:26 2003
***************
*** 339,351 ****
  	mkdir -p $(DATADIR)/camomile/locales
  	cp -f locales/*.mar $(DATADIR)/camomile/locales || true
  
! uninstall: $(UNINSTALL)
! 
  uninstall-without-ocamlfind:
! 	@echo "Uninstall not supported."
! 
  uninstall-with-ocamlfind:
  	ocamlfind remove camomile
  	rm -fr $(DATADIR)/camomile
  
  ###################################################################
--- 339,346 ----
  	mkdir -p $(DATADIR)/camomile/locales
  	cp -f locales/*.mar $(DATADIR)/camomile/locales || true
  
! uninstall: $(UNINSTALL) uninstall-data
!   
  uninstall-without-ocamlfind:
! 	rm -f $(OCAMLLIB)/camomile.*
! 	rm -f $(foreach f,$(INT) $(INT:cmi=cmx),$(subst 
public,$(OCAMLLIB),$(f)))
!   
  uninstall-with-ocamlfind:
  	ocamlfind remove camomile
+ 
+ uninstall-data:
  	rm -fr $(DATADIR)/camomile
  
  ###################################################################
